src/RobotCharacter.js

```diff
@@ -11,13 +11,11 @@ type RobotCharacterProps = {
 
 export default class RobotCharacter extends React.Component<RobotCharacterProps, {}> {
     render() {
-        const characterRadius = this.props.width / 2;
         const characterWidth = this.props.width * 0.75;
         return (
             <g
```

Comment:
If we no longer have the circle, can we get rid of this grouping element?

Reply:
Seems like applying transform directly to the icon is not working

Reply:
Ok, thanks for trying it.